torchreid/utils/torchtools.py

```diff
@@ -384,18 +384,13 @@ def load_pretrained_weights(model, file_path='', chkpt_name='model_weights', pre
         >>> file_path = 'log/my_model/model-best.pth.tar'
         >>> load_pretrained_weights(model, file_path)
     """
-    def _add_prefix(key, prefix):
-        key = prefix + "." + key
-        return key
-
     def _remove_prefix(key, prefix):
         prefix = prefix + '.'
         if key.startswith(prefix):
             key = key[len(prefix):]
         return key
 
     is_file = check_isfile(file_path)
-    pretraining_timm_model = model.pretrained and isinstance(model, TimmModelsWrapper)
```

Comment:
If we remove this - we can't load weights for timm models provided through link

Reply:
That depends on the link we provide. Downloading the official timm weights has no sense in this case: we will use custom link and custom snapshot with correct layer names.